src/main/java/core/basesyntax/FigureDrawer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawer {
```

Comment:
it's better to have a generic interface that could be reused for multiple cases, e.g. drawing not only figure.
```suggestion
public interface Drawer {
```